applications/inquiry_proposal_applicants/FoxGaming.md

```diff
@@ -0,0 +1,84 @@
+##### ***EGF Proposal Form (FoxGaming)***
+
+## ***Overview***
+
+**The project name**: Fox Gaming
+
+**Summary of your project and how it would help our ecosystem:** NFT-Marketplace is a must have every blockchain should have to attract new investors/users. Having it will increase transactions, users and  fee´s on the fx blockchain. 
+
+**Team Leadername:** Loil -Fox Gaming
+
+**Payment address:**  fx1780uflqpght7etzq8auv0ram6rwy3wd6dasex9 f(x)Core (Function X Mainnet), $FX
+
+**The level of amount you are requesting for? Level 1,2, 3, 4:** Level 3 50,000 USD in FX tokens
+
```

Comment:
kindly add in the details of the category you're applying this grant for. whether it is an NFT-marketplace or you have already built it but you are contracting artists to kick start it. depending on the answer, you might want to update other parts of the document too